app/ts/background/background.ts

```diff
@@ -301,8 +304,14 @@ async function handleSigningMode(
 		case 'eth_signTypedData_v3':
 		case 'eth_signTypedData_v4': return await personalSign(ethereumClientService, websiteTabConnections, socket, parsedRequest, request, false, website, settings)
 		case 'wallet_switchEthereumChain': return await switchEthereumChain(websiteTabConnections, socket, ethereumClientService, parsedRequest, request, false, website)
+		case 'eth_sendRawTransaction': {
+			if (isSupportedChain(settings.activeChain.toString()) ) {
+				return sendRawTransaction(ethereumClientService, parsedRequest, socket, request, false, website, settings)
+			}
```

Comment:
Should we be checking to see if the raw transaction's chainId matches the active chain's?